packages/twenty-front/src/modules/command-menu/pages/ask-ai/components/CommandMenuAskAIPage.tsx

```diff
@@ -21,17 +30,45 @@ export const CommandMenuAskAIPage = () => {
   const currentWorkspace = useRecoilValue(currentWorkspaceState);
   const agentId = currentWorkspace?.defaultAgent?.id;
 
-  if (!agentId) {
+  const [currentThreadId, setCurrentThreadId] = useRecoilComponentState(
+    currentAIChatThreadComponentState,
+    agentId,
+  );
+
+  const { loading: threadsLoading } = useGetAgentChatThreadsQuery({
+    variables: { agentId: agentId! },
+    skip: isDefined(currentThreadId) || !isDefined(agentId),
+    onCompleted: (data) => {
+      if (data.agentChatThreads.length > 0) {
+        setCurrentThreadId(data.agentChatThreads[0].id);
+      }
+    },
+  });
+
+  const { loading, data } = useGetAgentChatMessagesQuery({
+    variables: { threadId: currentThreadId as string },
```

Comment:
as needed here? If threadId is empty for some reason you should add a check for that here. Or if it's not possible that it's null/undefind then fix the root cause that led typescript to believe it was?